### ISSUE TITLE
dreadful

### ISSUE THREAD

**Issue by b3lief** [CPScript/Tools#2]
You claim to be ethical despite the fact that your GitHub README clearly states "I like Modding, Networking, Cracking, And Making Malware (MOST OF THEM ARE NOT AVAILABLE TO YOU 3)", which is not at all ethical. Not only can you not spell, but you misspelled illegal twice in the same area, and you generally post things that are unethical but claim to be ethical, which is very hypocritical.

**Comment by CPScript** [comment]
YES... you are correct its not ethical, BUT not illegal (TO MAKE THEM)... also I am ethical when it comes to using the repos that I make. I only make these because there are many online that are fake, and I really want people to be happy and get what they wanted... Im not trying to be hypocritical, I might not be the most ethical when I come to making, but I am when it comes to using. (just for you ill change my about me)... I HOPE THIS HELPS

**Comment by b3lief** [comment]
One thing you should not do is apologize all the time; it makes you appear weak and pathetic, and why should you change because of me? I was simply stating a fact that was illogical. It is your choice whether or not to change.

**Comment by CPScript** [comment]
i apologize to be respectful, also u were mainly correct... Im not only apologizing to you but to all programmers that would be offended by what i said, or by my opinion about myself. And i did change it because it was SLIGHTLY wrong... Also don't remember apologizing in the last comment that i made????
